apache/superset
# test: RTL testing on DatabaseModal

### SUMMARY
First draft of visual and functional testing for the DatabaseModal.

### TESTING INSTRUCTIONS
- Boot up your Superset development environment of choice
- cd into the `superset-frontend/src/views/CRUD/data/database/DatabaseModal` folder in your terminal
- Run `npm run test -t index.test.jsx`
- Observe the tests in the terminal

### ADDITIONAL INFORMATION
